### ISSUE TITLE
Collection error

### ISSUE THREAD

**Issue by HaoZeke** [lowmess/metalsmith-auto-collections#1]
When simply using

```js
ms.use(collect())
```

I get:

```bash
TypeError: Cannot read property 'pattern' of undefined
    at auto_collect (/home/haozeke/Github/Pandoc/Grimoire/node_modules/metalsmith-auto-collections/index.js:8:23)
    at metal (/home/haozeke/Github/Pandoc/Grimoire/gulpfile.babel.js:111:9)
    at taskWrapper (/home/haozeke/Github/Pandoc/Grimoire/node_modules/undertaker/lib/set-task.js:13:15)
    at bound (domain.js:301:14)
    at runBound (domain.js:314:12)
    at asyncRunner (/home/haozeke/Github/Pandoc/Grimoire/node_modules/async-done/index.js:39:18)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:655:11)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

Some of my files are not in subfolders..

**Comment by lowmess** [comment]
@HaoZeke Hey sorry it took me so long to get back to you! Looks like `auto-collections` currently expects an options object. I'm going to push out an update right now that will not error out the build, but in the meantime an empty object will resolve the fix:

```js
ms.use({})
```

**Comment by lowmess** [comment]
OK, just published `v0.1.2` that fixes the error, and properly captures all files if no pattern is sent. let me know if that works for you!

**Comment by HaoZeke** [comment]
Yup, works perfectly :)
